network.c

```diff
@@ -1193,6 +1193,38 @@ static ssize_t network_read_data(struct iio_context_pdata *pdata,
 	return network_recv(io_ctx, dst, len, 0);
 }
 
+static int network_consume(struct iio_context_pdata *pdata)
+{
+	int ret;
+	char dummy[1024];
```

Comment:
maybe we could reduce the buffer size to 128 or 64 bytes?
it means more reads, but 1k on the stack can be a lot;
some libcs limit the pthread stack size to 128k, so 1k can be a lot when writing an app with libiio;

Reply:
I guess I can reduce it for 512 but nowadays 1K should be more than acceptable. For instance on my system I have 8192k for the stack size by default. Even 32bit kernel (where stack occupation is really more "dangerous") have 1024 set for `Wframe-larger-than`

Reply:
glibc allocates 2M, 4M or 8M;
musl-libc 128k
uclibc-ng - don't know

it very much depends on which libc is used

Reply:
yes, I know... But even 128k is more than enough I guess. If an app is using the other 127K then I would argue that it's more on the app side than on the libiio. This feels like being really really cautious :smile: . But no strong feeling, I can reduce it to 256 as a compromise...

Reply:
well, the discussion is more like: someone will write a bad-app [anyway, it's pretty unavoidable], which uses plenty of stack, and it blows up in libiio and it ends up being on EZ [because that's where the stack blew up];
while, there are chances of happening with 256 / 128 / 64 bytes on the stack, i think it would be a higher chance with 1k than with less;

Reply:
Eheheh, you want to avoid EZ at all cost :smile:. I will jut give a bit more time to check if some other requests come up and then I will send a v2 with 256